netstandard/pkg/shims/netfx/System.Data.csproj

```diff
@@ -10,5 +8,5 @@
     <Compile Include="System.Data.Forwards.cs" />
     <ProjectReference Include="..\..\..\ref\netstandard.csproj" />
   </ItemGroup>
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
-</Project>
+	<Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.targets))\Directory.Build.targets" />
```

Comment:
Since you made these sdk projects you should remove these imports. I'm surprised this doesn't cause an issue.

Reply:
I kept these to get around some ordering issues - for example, in this case, if I remove the explicit import, then `Directory.Build.targets` gets imported too late for `OutDir` to be set (https://github.com/dotnet/standard/pull/886/files#diff-8c0a16ca2fc444198225ae58dd57d98fR40), but if I set it in `Directory.Build.props`, it's too early for it to not get overwritten by `ReferenceAssemblies.targets`